### PR TITLE
frontend load times monitoring with Sentry

### PR DESCRIPTION
**Pull request by neotyk**

### CODE REVIEW

File: src/cljs/athens/router.cljs [hunk 1]
```diff
@@ -63,25 +67,32 @@
...
-             :timeout  {:action :clear
-                        :id     :merge-prompt}
-             :dispatch [:editing/first-child (:block/uid page-block)]})
+            {:db         (-> db
+                             (assoc :current-route (assoc new-match :controllers controllers))
+                             (dissoc :merge-prompt))
+             :timeout    {:action :clear
+                          :id     :merge-prompt}
+             ;; TODO make it async flow
+             ;; halt? event trigger is when view is rendered
```
Comment (sid597) [inline src/cljs/athens/router.cljs:92]: Comment says TODO

**Review by sid597 — APPROVED** [review]
Did not review interceptors, hoc, macros because I think they are already reviewed by alex and filipe. Other than that LGTM, found just one todo comment.

File: src/cljs/athens/electron/boot.cljs [hunk 2]
```diff
@@ -74,13 +79,21 @@
                                                    [:navigate :home]
                                                    [:reset-undo-redo]
                                                    [:posthog/set-super-properties]
-                                                   [:loading/unset]]
-                                      ;; This event ends the async flow successfully.
-                                      :halt?      true}
+                                                   [:loading/unset]]}
+                                     {:when     :seen-all-of?
```
Comment (filipesilva) [inline src/cljs/athens/electron/boot.cljs:83]: Nit: this is a bit fragile, all it takes is for us to change one of these (or their flows) for the boot to fail mysteriously.

File: src/cljs/athens/views/pages/core.cljs [hunk 3]
```diff
@@ -30,23 +31,6 @@
                                                 :border-radius "0.5rem"}}})
 
 
-;; Helpers
-
-
-(defn match-page
```
Comment (filipesilva) [inline src/cljs/athens/views/pages/core.cljs:36]: Nit: odd inlining, docstring was lost in the process.

**Review by filipesilva — APPROVED** [review]

**Pull request merged**